Feature/display products

Sakshi-Simform opened the pull request:
# Description
- create ui to display post from api
- create crud operation like add, update ,delete

# Type of change
- [ ] Bug fix
- [x] New feature
- [ ] Documentation update
- [ ] Refactoring
- [ ] Performance improvement
- [ ] Scaffolding

# Screenshots
![Screenshot from 2025-03-31 19-23-25](https://github.com/user-attachments/assets/fa0b1c76-9bce-4304-a892-0cadbdcd67ff)
![Screenshot from 2025-04-09 16-10-27](https://github.com/user-attachments/assets/73982258-1048-4310-b1a8-c575e8555ac8)
![Screenshot from 2025-04-09 15-59-15](https://github.com/user-attachments/assets/3ca7a0dc-34e6-4346-8562-e2bbd47101ea)
![Screenshot from 2025-04-01 15-26-38](https://github.com/user-attachments/assets/c632d763-b7fc-4335-8c1f-2089fab50a95)
![Screenshot from 2025-04-09 16-10-13](https://github.com/user-attachments/assets/a2b78418-80bc-459b-a6ad-ed768969a9ee)



# How to test
https://deploy-preview-3--crud-productm.netlify.app/

dhruv-d-simform commented on src/script/storage.ts:
```diff
@@ -0,0 +1,28 @@
+import { Post } from './api.ts';
```
We can use `type` here to optimize this. 
```suggestion
import type { Post } from './api.ts';
```

Sakshi-Simform replied:
add https://github.com/Sakshi-Simform/product-management/pull/3/commits/9c5516df96f609dd01b52a749d2f3637e6cff67b

dhruv-d-simform commented on src/script/ui.ts:
```diff
@@ -0,0 +1,26 @@
+import { Post } from "./api";
```
We can use `type` here to optimize this. 
```suggestion
import type { Post } from "./api";
```

Sakshi-Simform replied:
add https://github.com/Sakshi-Simform/product-management/pull/3/commits/9c5516df96f609dd01b52a749d2f3637e6cff67b

dhruvdhaduk-simform commented on src/script/eventhandler.ts:
```diff
@@ -0,0 +1,104 @@
+
+// Search function to filter posts
+const searchPosts = () => {
+    console.log("hello")
```
We can remove this console log

dhruv-d-simform commented on src/script/eventhandler.ts:
```diff
@@ -0,0 +1,104 @@
+window.onload = loadPosts;
+saveButton.addEventListener('click', addOrUpdatePost);
+
+
```
We can remove unnecessary empty lines from here.

Sakshi-Simform replied:
remove https://github.com/Sakshi-Simform/product-management/pull/3/commits/b76d7ebf024c212811b41b960f24861ee41fde25

Sakshi-Simform replied:
removed https://github.com/Sakshi-Simform/product-management/pull/3/commits/b76d7ebf024c212811b41b960f24861ee41fde25

krushna-simform commented:
Can we change change the project name because you are loading post data not product data! let me know your thoughts.

krushna-simform commented:
Can we align this button at the bottom.

![Screenshot from 2025-04-01 13-52-44](https://github.com/user-attachments/assets/ffd779a3-8235-4855-924e-dffe3b3989c7)

Sakshi-Simform commented:
> Can we align this button at the bottom.
> 
> ![Screenshot from 2025-04-01 13-52-44](https://private-user-images.githubusercontent.com/194056780/428938344-ffd779a3-8235-4855-924e-dffe3b3989c7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDM0OTgxOTMsIm5iZiI6MTc0MzQ5Nzg5MywicGF0aCI6Ii8xOTQwNTY3ODAvNDI4OTM4MzQ0LWZmZDc3OWEzLTgyMzUtNDg1NS05MjRlLWRmZmUzYjM5ODljNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNDAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDQwMVQwODU4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iODlhOWExM2ViMjgxNmM2ODFlOGYwNGE1NTUxOTljYjQ4ZWE1YTNmYjUwN2MzNDZjZmU4YjUyOGI2YTAwNmZlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.UjDTFoxt4Y3R52dN2lYMX7MHvrf-VUWSx7yXp8S1Cug)

https://github.com/Sakshi-Simform/product-management/pull/3/commits/80824acfa6af10fb30b546a99bb4a97c534472c3 added

Sakshi-Simform commented:
> Can we change change the project name because you are loading post data not product data! let me know your thoughts.

https://github.com/Sakshi-Simform/product-management/pull/3/commits/d46a3ec408446675e9a8176c149a767a01e0bf77 change name of project

krushna-simform commented:
Can we add one `alert` message when user press `delete` button ?

like `Are you sure you want to delete this user?`

Sakshi-Simform commented:
> Can we add one `alert` message when user press `delete` button ?
> 
> like `Are you sure you want to delete this user?`

add 

> Can we add one `alert` message when user press `delete` button ?
> 
> like `Are you sure you want to delete this user?`

add https://github.com/Sakshi-Simform/product-management/pull/3/commits/7fd217e28095a74101bb515ae2fb08b11279ce61

krushna-simform commented on src/script/api.ts:
```diff
@@ -0,0 +1,37 @@
+}
+
+// Fetch posts from API
+export const fetchPosts = async (): Promise<Post[]> => {
```
can we use `Array<Post>`.

```suggestion
export const fetchPosts = async (): Promise<Array<post>> => {
```

Sakshi-Simform replied:
change https://github.com/Sakshi-Simform/product-management/commit/0ad89dad04323d8da0a14f856deb28e24eef7889

krushna-simform commented on src/script/debounce.ts:
```diff
@@ -0,0 +1,12 @@
+    func: T,
+    delay: number
+): (...args: Parameters<T>) => void {
+    let timer: ReturnType<typeof setTimeout> | null = null;
```
```suggestion
    let timer: ReturnType<typeof setTimeout> | undefined;
```

Sakshi-Simform replied:
change https://github.com/Sakshi-Simform/product-management/commit/0ad89dad04323d8da0a14f856deb28e24eef7889

krushna-simform commented on src/script/debounce.ts:
```diff
@@ -0,0 +1,12 @@
+    delay: number
+): (...args: Parameters<T>) => void {
+    let timer: ReturnType<typeof setTimeout> | null = null;
+    return (...args: Parameters<T>) => {
```
```suggestion
    return function(this: ThisParameterType<T>,...args: Parameters<T>) => {
```

Sakshi-Simform replied:
change https://github.com/Sakshi-Simform/product-management/commit/0ad89dad04323d8da0a14f856deb28e24eef7889

krushna-simform commented on src/script/debounce.ts:
```diff
@@ -0,0 +1,12 @@
+): (...args: Parameters<T>) => void {
+    let timer: ReturnType<typeof setTimeout> | null = null;
+    return (...args: Parameters<T>) => {
+        if (timer) clearTimeout(timer);
```
```suggestion
        if (timer !== undefined) clearTimeout(timer);
```

Sakshi-Simform replied:
change https://github.com/Sakshi-Simform/product-management/commit/0ad89dad04323d8da0a14f856deb28e24eef7889

krushna-simform commented on src/script/debounce.ts:
```diff
@@ -0,0 +1,12 @@
+    return (...args: Parameters<T>) => {
+        if (timer) clearTimeout(timer);
+        timer = setTimeout(() => {
+            func(...args);
```
```suggestion
            func.apply(this, args);
```

Sakshi-Simform replied:
change https://github.com/Sakshi-Simform/product-management/pull/3/commits/0ad89dad04323d8da0a14f856deb28e24eef7889

krushna-simform commented on src/script/storage.ts:
```diff
@@ -0,0 +1,28 @@
+import type{ Post } from './api.ts';
+
+export const getLocalPosts = (): Post[] => {
```
```suggestion
export const getLocalPosts = (): Array<Post> => {
```

Sakshi-Simform replied:
change https://github.com/Sakshi-Simform/product-management/pull/3/commits/0ad89dad04323d8da0a14f856deb28e24eef7889

krushna-simform replied:
Can you add appropriate Commit id's

krushna-simform commented on src/script/storage.ts:
```diff
@@ -0,0 +1,28 @@
+    return JSON.parse(localStorage.getItem('posts') || '[]');
+};
+
+export const storePostsLocally = (posts: Post[]) => {
```
```suggestion
export const storePostsLocally = (posts: Array<Post>) => {
```

Sakshi-Simform replied:
add https://github.com/Sakshi-Simform/product-management/commit/7492e5b323acd1224e5f15882dc71b2ed994b8fa

Kaushal-SF replied:
@Sakshi-Simform Have we accommodate this change?

add https://github.com/Sakshi-Simform/product-management/pull/3/commits/727aaa2ce2a7f5f3ae7f34860e55e9fd1e24c018

krushna-simform commented on src/script/ui.ts:
```diff
@@ -0,0 +1,26 @@
+) as HTMLDivElement;
+
+// Display posts dynamically
+export const displayPosts = (posts: Post[]) => {
```
```suggestion
export const displayPosts = (posts: Array<Post>) => {
```

Sakshi-Simform replied:
add https://github.com/Sakshi-Simform/product-management/pull/3/commits/7492e5b323acd1224e5f15882dc71b2ed994b8fa

Kaushal-SF commented:
@Sakshi-Simform Overall UI is looking good. There is one thing that we can improve.

When we don't find any post, we can provide a message instead of empty screen.
<img width="500" alt="image" src="https://github.com/user-attachments/assets/9133220b-817d-48be-b7cb-bda6899e8d68" />

Kaushal-SF commented:
@Sakshi-Simform Have we figured out the reason for comparatively low score for `Accessibility`? What are the reasons for that?

<img width="500" alt="image" src="https://github.com/user-attachments/assets/8341157a-4212-4c29-99e7-e2f4eb24072b" />

Kaushal-SF commented:
@Sakshi-Simform, have we check the UI with this platform? 
- https://wave.webaim.org/

Kaushal-SF commented on src/script/ui.ts:
Please provide a meaningful file name. `ui.ts` is not making much sense here.

Sakshi-Simform replied:
change file name

Kaushal-SF commented on src/script/api.ts:
```diff
@@ -0,0 +1,37 @@
+    id: number;
+    title: string;
+    body: string;
+}
```
Nit: can provide a meaningful name for interface.
```suggestion
export interface PostData {
    id: number;
    title: string;
    body: string;
}
```

Sakshi-Simform replied:
chnage https://github.com/Sakshi-Simform/product-management/pull/3/commits/d563ce1906f37594112f1f659420ebbaef655ce4

Kaushal-SF commented on src/script/api.ts:
```diff
@@ -0,0 +1,37 @@
+    if (!hasMore || isFetching) return [];
+    isFetching = true;
+    const response = await fetch(`${API_URL}?limit=${LIMIT}&skip=${skip}`);
+    const data = await response.json();
```
While fetching the data, it is a good idea to use `try and catch`.

Sakshi-Simform replied:
add https://github.com/Sakshi-Simform/product-management/pull/3/commits/d7429df310c96097771d67c7b696e91d01787c27

Kaushal-SF commented on src/script/api.ts:
```diff
@@ -0,0 +1,37 @@
+};
+
+// Update post in API
+export const updatePostAPI = async (post: Post) => {
```
```suggestion
export const updatePost = async (post: Post) => {
```

Sakshi-Simform replied:
change https://github.com/Sakshi-Simform/product-management/pull/3/commits/286ec146367b5079a8a6e61dec56ccb47a44d455

Kaushal-SF commented on src/script/api.ts:
```diff
@@ -0,0 +1,37 @@
+};
+
+// Delete post from API
+export const deletePostAPI = async (id: number) => {
```
```suggestion
export const deletePost= async (id: number) => {
```

Sakshi-Simform replied:
change https://github.com/Sakshi-Simform/product-management/commit/286ec146367b5079a8a6e61dec56ccb47a44d455  https://github.com/Sakshi-Simform/product-management/pull/3/commits/652f7aa13200a53c4aacb83f18f54a5e60b19acf

Kaushal-SF commented on src/script/ui.ts:
```diff
@@ -0,0 +1,26 @@
+import type{ Post } from "./api";
+
+const postContainer = document.getElementById(
```
we can check whether `postContainer` is available or not?

Sakshi-Simform replied:
add https://github.com/Sakshi-Simform/product-management/pull/3/commits/ec7c53d7a1625112347e95c69b255cd0131c46e7

Kaushal-SF commented on src/script/ui.ts:
```diff
@@ -0,0 +1,26 @@
+        const postElement = document.createElement('div');
+        postElement.className = 'post';
+        postElement.setAttribute('data-id', post.id.toString());
+        postElement.innerHTML = `
```
While using `innerHTML` we make sure that what are the disadvantages it brings to the website.

Sakshi-Simform replied:
for this do I need to change anything?

Kaushal-SF replied:
@Sakshi-Simform Leaving up to you if you want to make changes or not.

working on it

fixed https://github.com/Sakshi-Simform/product-management/pull/3/commits/9604a622c439bebb26bc4b69f97070bf0953b862

Kaushal-SF commented on src/script/storage.ts:
```diff
@@ -0,0 +1,28 @@
+
+export const getLocalPosts = (): Array<Post> => {
+    return JSON.parse(localStorage.getItem('posts') || '[]');
+};
```
Are we sure we want this `'[]'` as an alternative value?

Sakshi-Simform replied:
Yes I want to return empty array if dont get any value

Kaushal-SF replied:
@Sakshi-Simform What is the need to use quotes here? If you see the above comment, I pointed that you have used `'[]'` instead of `[]`. Was that a correct one?

fixed https://github.com/Sakshi-Simform/product-management/pull/3/commits/42b622070e712936335a41ecbd74ed3eaca2df75

Kaushal-SF replied:
@Sakshi-Simform ^^^
What about this comment??

Sakshi-Simform replied:
fixed fixed https://github.com/Sakshi-Simform/product-management/commit/63cdf39990eb874f6c7375db053f54bb91ae27c5

Sakshi-Simform commented:
> @Sakshi-Simform Have we figured out the reason for comparatively low score for `Accessibility`? What are the reasons for that?
> 
> <img alt="image" width="500" src="https://private-user-images.githubusercontent.com/97082607/431205534-8341157a-4212-4c29-99e7-e2f4eb24072b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQwODg0OTUsIm5iZiI6MTc0NDA4ODE5NSwicGF0aCI6Ii85NzA4MjYwNy80MzEyMDU1MzQtODM0MTE1N2EtNDIxMi00YzI5LTk5ZTctZTJmNGViMjQwNzJiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDA4VDA0NTYzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMzYWEzZTM3ZGRmMjcyYWMyY2RmODA4OTY1NmU4ZjIxNWIwMWM0OTMzYzBhNTdjODZkMmQzZDIyNDU4YmY3OWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.YHcuoqy_c-QnUJhM7r8lAj-Wgi4jC8ihTziOAZbslig">

Yes that is coming due to aspect ratio

Kaushal-SF commented on src/script/main.ts:
We can omit this change as we don't have much thing to export from this file.

Sakshi-Simform replied:
removed https://github.com/Sakshi-Simform/product-management/pull/3/commits/3587f6b375814d2c825410456984dc3e23399b76

Sakshi-Simform commented:
> @Sakshi-Simform, have we check the UI with this platform?
> 
> * https://wave.webaim.org/

added

Kaushal-SF commented on src/script/eventhandler.ts:
```diff
@@ -0,0 +1,122 @@
+    const scrollPosition = window.scrollY + window.innerHeight;
+    const pageHeight = document.documentElement.scrollHeight;
+
+    if (scrollPosition >= pageHeight - 100) {
+        isFetching = true;
```
Can take this number into one constant and probably can increase it to `150`.

Sakshi-Simform replied:
add https://github.com/Sakshi-Simform/product-management/pull/3/commits/cdaedfa4a8e3f698f7bce97cffb856c0eddda84d

Kaushal-SF reviewed (commented):
@Sakshi-Simform Added few comments for code improvements. Otherwise code looks good.

Sakshi-Simform commented:
> @Sakshi-Simform Overall UI is looking good. There is one thing that we can improve.
> 
> When we don't find any post, we can provide a message instead of empty screen. <img alt="image" width="500" src="https://private-user-images.githubusercontent.com/97082607/431202721-9133220b-817d-48be-b7cb-bda6899e8d68.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQwOTk0ODAsIm5iZiI6MTc0NDA5OTE4MCwicGF0aCI6Ii85NzA4MjYwNy80MzEyMDI3MjEtOTEzMzIyMGItODE3ZC00OGJlLWI3Y2ItYmRhNjg5OWU4ZDY4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDA4VDA3NTk0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIxOWExZWMxZGZiNjY2OTk4YWFiZWFkZWRhZTNjMTE0MGQyOTIxZTRmMDQ0ZWM3MmUxZDlmYjA0ZWJmMDE5MDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.rWlFkiJNu6I7MsR_c3nZFcjV9JQrE-SVwmYRa4WmrSo">

add https://github.com/Sakshi-Simform/product-management/pull/3/commits/af25798e42587627b8888d3c2e64ba1d528c591f

dhruv-d-simform commented:
![Screenshot from 2025-04-08 14-02-11](https://github.com/user-attachments/assets/a13ffcc2-a1aa-4259-a8cc-eea1d3fef20d)

Can we handle this overflow ?

Sakshi-Simform commented:
> ![Screenshot from 2025-04-08 14-02-11](https://private-user-images.githubusercontent.com/194202800/431281038-a13ffcc2-a1aa-4259-a8cc-eea1d3fef20d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQxMDIxMDQsIm5iZiI6MTc0NDEwMTgwNCwicGF0aCI6Ii8xOTQyMDI4MDAvNDMxMjgxMDM4LWExM2ZmY2MyLWExYWEtNDI1OS1hOGNjLWVlYTFkM2ZlZjIwZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNDA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDQwOFQwODQzMjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNzE5Zjc1MzIxNGI1YWMxNTFjOGU0MmUzMzJkYTc4NWUyM2RiZGFjN2I2MzIwYjAxNGQ1N2U2MmExZDYzOGU5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ousR6Qk1EpVMd5pL_gydDUJA89NX6hX9aNTPC8Ja97g)
> 
> Can we handle this overflow ?

solved https://github.com/Sakshi-Simform/product-management/pull/3/commits/cf64263adb74f016779af7ee96dcfcd1b92b87a1 https://github.com/Sakshi-Simform/product-management/pull/3/commits/878923f5142046e6b98b5f23ff73f71e9b2822a2

krushna-simform commented:
This buttons are also scrolling. Can we handle this case? 

- `hint`: You can provide character limit to `title` and `description` so you don't need to handle overflow case.

![Screenshot from 2025-04-08 14-40-42](https://github.com/user-attachments/assets/3c8ce9f3-6739-4d38-a194-9efce18c4565)

krushna-simform commented:
> This buttons are also scrolling. Can we handle this case?
> 
> * `hint`: You can provide character limit to `title` and `description` so you don't need to handle overflow case.
> 
> ![Screenshot from 2025-04-08 14-40-42](https://private-user-images.githubusercontent.com/194056780/431297167-3c8ce9f3-6739-4d38-a194-9efce18c4565.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQxMDQxNjYsIm5iZiI6MTc0NDEwMzg2NiwicGF0aCI6Ii8xOTQwNTY3ODAvNDMxMjk3MTY3LTNjOGNlOWYzLTY3MzktNGQzOC1hMTk0LTllZmNlMThjNDU2NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNDA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDQwOFQwOTE3NDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNzA3NGU4YWIxMGIyOTZhNjMzNzAzNThlODA5NmY5NmJlOTUxMWIwMDI3YThjMDJkMzNlYWEzN2EzOTJhNjg0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.z8UZNnXmA3uRFJ1R1DCtAd0nBGw---VjeOrGmaXh36o)

can we provide inline error for character limit instead of alert box.

Sakshi-Simform commented:
> > This buttons are also scrolling. Can we handle this case?
> > 
> > * `hint`: You can provide character limit to `title` and `description` so you don't need to handle overflow case.
> > 
> > ![Screenshot from 2025-04-08 14-40-42](https://private-user-images.githubusercontent.com/194056780/431297167-3c8ce9f3-6739-4d38-a194-9efce18c4565.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQxMDQxNjYsIm5iZiI6MTc0NDEwMzg2NiwicGF0aCI6Ii8xOTQwNTY3ODAvNDMxMjk3MTY3LTNjOGNlOWYzLTY3MzktNGQzOC1hMTk0LTllZmNlMThjNDU2NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNDA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDQwOFQwOTE3NDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNzA3NGU4YWIxMGIyOTZhNjMzNzAzNThlODA5NmY5NmJlOTUxMWIwMDI3YThjMDJkMzNlYWEzN2EzOTJhNjg0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.z8UZNnXmA3uRFJ1R1DCtAd0nBGw---VjeOrGmaXh36o)
> 
> can we provide inline error for character limit instead of alert box.

add https://github.com/Sakshi-Simform/product-management/pull/3/commits/bb3959f95d368c3570ebb366bdf0acc2d9ccdb23 https://github.com/Sakshi-Simform/product-management/pull/3/commits/14227268a462f47e24f3284a1346bae9ca8840b6

Kaushal-SF commented:
> Yes that is coming due to aspect ratio

@Sakshi-Simform I think you overlooked the result. I generated the report and there are other issue other than `aspect ratio`.

<img width="500" alt="image" src="https://github.com/user-attachments/assets/9f047656-d723-4441-912b-022ff9587363" />

Kaushal-SF commented:
> > @Sakshi-Simform, have we check the UI with this platform?
> > 
> > * https://wave.webaim.org/
> 
> added

@Sakshi-Simform By testing our UI with this platform, our goal is to figure what errors it is providing and resolve them. Can you see any errors in the generated report?

Kaushal-SF commented on src/script/displaypost.ts:
Use `Camel Case` for file name. e.g., `displayPost.ts`

Sakshi-Simform replied:
change https://github.com/Sakshi-Simform/product-management/pull/3/commits/c19325b269e5052f415f60474fb979d982bc5c5a

Kaushal-SF commented:
@Sakshi-Simform No validation for title and desc. It is also accepting just numbers. That shouldn't be the case. It should allow the mix of character.

<img width="241" alt="image" src="https://github.com/user-attachments/assets/76e18093-b5b8-4608-b150-b7310076a284" />

Kaushal-SF commented:
@Sakshi-Simform We can add a `loader` to provide an indicator that posts are loading. Currently, we are not showing anything.

Sakshi-Simform commented:
> @Sakshi-Simform No validation for title and desc. It is also accepting just numbers. That shouldn't be the case. It should allow the mix of character.
> 
> <img alt="image" width="241" src="https://private-user-images.githubusercontent.com/97082607/431656327-76e18093-b5b8-4608-b150-b7310076a284.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQxOTIzODksIm5iZiI6MTc0NDE5MjA4OSwicGF0aCI6Ii85NzA4MjYwNy80MzE2NTYzMjctNzZlMTgwOTMtYjViOC00NjA4LWIxNTAtYjczMTAwNzZhMjg0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDA5VDA5NDgwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA3Mjk1MjZhMmNlNzg5YWJkNTgzYTFlMjU2NWM1N2RmYjNlYzdjYTlkZWViM2MyYTcyNDE2ZWM3OWMwY2E5NDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.AGd_0zfGJThFMZvnI2YLiEpxpPSYASe93jVmcBhZLxY">

added https://github.com/Sakshi-Simform/product-management/pull/3/commits/ff9a4e4af95413853ce2520b935f8b0c55451f1d

Sakshi-Simform commented:
> > > @Sakshi-Simform, have we check the UI with this platform?
> > > 
> > > * https://wave.webaim.org/
> > 
> > 
> > added
> 
> @Sakshi-Simform By testing our UI with this platform, our goal is to figure what errors it is providing and resolve them. Can you see any errors in the generated report?

solved

Sakshi-Simform commented:
> > Yes that is coming due to aspect ratio
> 
> @Sakshi-Simform I think you overlooked the result. I generated the report and there are other issue other than `aspect ratio`.
> 
> <img alt="image" width="500" src="https://private-user-images.githubusercontent.com/97082607/431654178-9f047656-d723-4441-912b-022ff9587363.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQxOTU0MjQsIm5iZiI6MTc0NDE5NTEyNCwicGF0aCI6Ii85NzA4MjYwNy80MzE2NTQxNzgtOWYwNDc2NTYtZDcyMy00NDQxLTkxMmItMDIyZmY5NTg3MzYzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDA5VDEwMzg0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM3YzFkYTFjNTQ0NjEyNTU5ODg3NDNmOWQ3NTNhYTg0NmE1N2MzZDlmYzAzZjIyODhlNmVlMmMxODUzZjU0YjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.gfssHbSB5rDs5GvTE_XZKufEwC9JfmCUaDVW-mIfpR8">

solved

Sakshi-Simform commented:
> > > This buttons are also scrolling. Can we handle this case?
> > > 
> > > * `hint`: You can provide character limit to `title` and `description` so you don't need to handle overflow case.
> > > 
> > > ![Screenshot from 2025-04-08 14-40-42](https://private-user-images.githubusercontent.com/194056780/431297167-3c8ce9f3-6739-4d38-a194-9efce18c4565.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQxMDQxNjYsIm5iZiI6MTc0NDEwMzg2NiwicGF0aCI6Ii8xOTQwNTY3ODAvNDMxMjk3MTY3LTNjOGNlOWYzLTY3MzktNGQzOC1hMTk0LTllZmNlMThjNDU2NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNDA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDQwOFQwOTE3NDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNzA3NGU4YWIxMGIyOTZhNjMzNzAzNThlODA5NmY5NmJlOTUxMWIwMDI3YThjMDJkMzNlYWEzN2EzOTJhNjg0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.z8UZNnXmA3uRFJ1R1DCtAd0nBGw---VjeOrGmaXh36o)
> > 
> > 
> > can we provide inline error for character limit instead of alert box.
> 
> add [bb3959f](https://github.com/Sakshi-Simform/product-management/commit/bb3959f95d368c3570ebb366bdf0acc2d9ccdb23) [1422726](https://github.com/Sakshi-Simform/product-management/commit/14227268a462f47e24f3284a1346bae9ca8840b6)

![Screenshot from 2025-04-09 16-14-10](https://github.com/user-attachments/assets/9949362c-270e-472c-b670-54e69964af38)
solved

Sakshi-Simform commented:
> > > > This buttons are also scrolling. Can we handle this case?
> > > > 
> > > > * `hint`: You can provide character limit to `title` and `description` so you don't need to handle overflow case.
> > > > 
> > > > ![Screenshot from 2025-04-08 14-40-42](https://private-user-images.githubusercontent.com/194056780/431297167-3c8ce9f3-6739-4d38-a194-9efce18c4565.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQxMDQxNjYsIm5iZiI6MTc0NDEwMzg2NiwicGF0aCI6Ii8xOTQwNTY3ODAvNDMxMjk3MTY3LTNjOGNlOWYzLTY3MzktNGQzOC1hMTk0LTllZmNlMThjNDU2NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNDA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDQwOFQwOTE3NDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNzA3NGU4YWIxMGIyOTZhNjMzNzAzNThlODA5NmY5NmJlOTUxMWIwMDI3YThjMDJkMzNlYWEzN2EzOTJhNjg0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.z8UZNnXmA3uRFJ1R1DCtAd0nBGw---VjeOrGmaXh36o)
> > > 
> > > 
> > > can we provide inline error for character limit instead of alert box.
> > 
> > 
> > add [bb3959f](https://github.com/Sakshi-Simform/product-management/commit/bb3959f95d368c3570ebb366bdf0acc2d9ccdb23) [1422726](https://github.com/Sakshi-Simform/product-management/commit/14227268a462f47e24f3284a1346bae9ca8840b6)
> 
> ![Screenshot from 2025-04-09 16-14-10](https://private-user-images.githubusercontent.com/194056200/431802350-9949362c-270e-472c-b670-54e69964af38.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQxOTU4MTUsIm5iZiI6MTc0NDE5NTUxNSwicGF0aCI6Ii8xOTQwNTYyMDAvNDMxODAyMzUwLTk5NDkzNjJjLTI3MGUtNDcyYy1iNjcwLTU0ZTY5OTY0YWYzOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNDA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDQwOVQxMDQ1MTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YTQxZTkzYjJjMGFhZDk0YjVlMDRmZGI1N2Y5YWNkYjM0MTdhMjdjMWIxYmEzZTQyYjA0MzkxZDI0Mjk4MWUyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Vj128sNfupq--M8e7feRKraR3bmniciEi0fR43gwVok) solved

![Screenshot from 2025-04-09 16-13-41](https://github.com/user-attachments/assets/3a99b0ca-fd98-4e49-b04b-6d6b27e85f90)
added

Kaushal-SF commented:
@Sakshi-Simform I think we haven't implemented a loader yet.

One more thing, I just noticed. When we reach to the end of post (no more posts to fetch). And if we click on `Edit`, we need to come up at the top to the page and edit the details.

It would be better if we can scroll-up to the top if user is clicking on `Edit` button.

Sakshi-Simform commented:
> @Sakshi-Simform I think we haven't implemented a loader yet.
> 
> One more thing, I just noticed. When we reach to the end of post (no more posts to fetch). And if we click on `Edit`, we need to come up at the top to the page and edit the details.
> 
> It would be better if we can scroll-up to the top if user is clicking on `Edit` button.

yes working for loader

Sakshi-Simform commented:
> @Sakshi-Simform I think we haven't implemented a loader yet.
> 
> One more thing, I just noticed. When we reach to the end of post (no more posts to fetch). And if we click on `Edit`, we need to come up at the top to the page and edit the details.
> 
> It would be better if we can scroll-up to the top if user is clicking on `Edit` button.

add https://github.com/Sakshi-Simform/product-management/pull/3/commits/05bbd445e09472580098dca567cdecf30b1ce2d9

Sakshi-Simform commented:
> @Sakshi-Simform We can add a `loader` to provide an indicator that posts are loading. Currently, we are not showing anything.

add https://github.com/Sakshi-Simform/product-management/pull/3/commits/05bbd445e09472580098dca567cdecf30b1ce2d9

Kaushal-SF commented:
@Sakshi-Simform For Loader, we can perhaps make the border color bit darker for better visibility.

<img width="215" alt="image" src="https://github.com/user-attachments/assets/ce7beb5e-a02a-4f53-9d2c-04211206c83b" />

Kaushal-SF reviewed (approved):
LGTM! Added minor comment.

Kaushal-SF commented:
@dhruvdhaduk-simform Your review is pending. Please review and approve the PR.

Kaushal-SF commented:
@Sakshi-Simform Noticed that there are few previous comments which are yet to resolve. 

JFYI, make sure you have addressed all the feedback provided and made the necessary changes.

Sakshi-Simform commented:
> @Sakshi-Simform For Loader, we can perhaps make the border color bit darker for better visibility.
> 
> <img alt="image" width="215" src="https://private-user-images.githubusercontent.com/97082607/432596350-ce7beb5e-a02a-4f53-9d2c-04211206c83b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQzNDg3NDksIm5iZiI6MTc0NDM0ODQ0OSwicGF0aCI6Ii85NzA4MjYwNy80MzI1OTYzNTAtY2U3YmViNWUtYTAyYS00ZjUzLTlkMmMtMDQyMTEyMDZjODNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDExVDA1MTQwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU0NjUzMjA1N2Q0YWZmY2IyZTFmZDVhZTQ1MWQ5ZGU3NTcwOTk0ZWRjZDAwOGFhMDQ4ODBmNDZhMDc3ZDA4NzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.qiJTUlHbcpsIDuzD6JVUkb_2TJ-1kpnCKpW346iyoic">

change https://github.com/Sakshi-Simform/product-management/pull/3/commits/48f09483f0b0691c54673943203e30ccaec58ad9

Kaushal-SF commented:
> > <img alt="image" width="215" src="https://private-user-images.githubusercontent.com/97082607/432596350-ce7beb5e-a02a-4f53-9d2c-04211206c83b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQzNDg3NDksIm5iZiI6MTc0NDM0ODQ0OSwicGF0aCI6Ii85NzA4MjYwNy80MzI1OTYzNTAtY2U3YmViNWUtYTAyYS00ZjUzLTlkMmMtMDQyMTEyMDZjODNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDExVDA1MTQwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU0NjUzMjA1N2Q0YWZmY2IyZTFmZDVhZTQ1MWQ5ZGU3NTcwOTk0ZWRjZDAwOGFhMDQ4ODBmNDZhMDc3ZDA4NzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.qiJTUlHbcpsIDuzD6JVUkb_2TJ-1kpnCKpW346iyoic">
> 
> change [48f0948](https://github.com/Sakshi-Simform/product-management/commit/48f09483f0b0691c54673943203e30ccaec58ad9)

@Sakshi-Simform It is not working properly. Can you please check it again?

Sakshi-Simform commented:
> > > <img alt="image" width="215" src="https://private-user-images.githubusercontent.com/97082607/432596350-ce7beb5e-a02a-4f53-9d2c-04211206c83b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQzNDg3NDksIm5iZiI6MTc0NDM0ODQ0OSwicGF0aCI6Ii85NzA4MjYwNy80MzI1OTYzNTAtY2U3YmViNWUtYTAyYS00ZjUzLTlkMmMtMDQyMTEyMDZjODNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDExVDA1MTQwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU0NjUzMjA1N2Q0YWZmY2IyZTFmZDVhZTQ1MWQ5ZGU3NTcwOTk0ZWRjZDAwOGFhMDQ4ODBmNDZhMDc3ZDA4NzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.qiJTUlHbcpsIDuzD6JVUkb_2TJ-1kpnCKpW346iyoic">
> > 
> > 
> > change [48f0948](https://github.com/Sakshi-Simform/product-management/commit/48f09483f0b0691c54673943203e30ccaec58ad9)
> 
> @Sakshi-Simform It is not working properly. Can you please check it again?

I'll check that

Sakshi-Simform commented:
> > > > <img alt="image" width="215" src="https://private-user-images.githubusercontent.com/97082607/432596350-ce7beb5e-a02a-4f53-9d2c-04211206c83b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQzNDg3NDksIm5iZiI6MTc0NDM0ODQ0OSwicGF0aCI6Ii85NzA4MjYwNy80MzI1OTYzNTAtY2U3YmViNWUtYTAyYS00ZjUzLTlkMmMtMDQyMTEyMDZjODNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDExVDA1MTQwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU0NjUzMjA1N2Q0YWZmY2IyZTFmZDVhZTQ1MWQ5ZGU3NTcwOTk0ZWRjZDAwOGFhMDQ4ODBmNDZhMDc3ZDA4NzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.qiJTUlHbcpsIDuzD6JVUkb_2TJ-1kpnCKpW346iyoic">
> > > 
> > > 
> > > change [48f0948](https://github.com/Sakshi-Simform/product-management/commit/48f09483f0b0691c54673943203e30ccaec58ad9)
> > 
> > 
> > @Sakshi-Simform It is not working properly. Can you please check it again?
> 
> I'll check that

now that is working

Kaushal-SF commented:
> > > @Sakshi-Simform It is not working properly. Can you please check it again?
> > 
> > 
> > I'll check that
> 
> now that is working

@Sakshi-Simform
- Go to your latest build, open it.
- Throttle the network to 3G or 4g.
- Check the loader. Is it looking the same as I provided a screenshot above.

Sakshi-Simform commented:
> > > > @Sakshi-Simform It is not working properly. Can you please check it again?
> > > 
> > > 
> > > I'll check that
> > 
> > 
> > now that is working
> 
> @Sakshi-Simform
> 
> * Go to your latest build, open it.
> * Throttle the network to 3G or 4g.
> * Check the loader. Is it looking the same as I provided a screenshot above.

![Screenshot from 2025-04-11 13-30-36](https://github.com/user-attachments/assets/876ec99b-e5e2-4e5a-9b92-41039ef7f6dd)
it is working

Kaushal-SF commented:
@Sakshi-Simform Have you compared your UI with the screenshot that I have provided? Just compare the UI. Can you see any difference?

Please check this comment again: https://github.com/Sakshi-Simform/product-management/pull/3#issuecomment-2795857460

<img alt="image" width="215" src="https://private-user-images.githubusercontent.com/97082607/432596350-ce7beb5e-a02a-4f53-9d2c-04211206c83b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQzNDg3NDksIm5iZiI6MTc0NDM0ODQ0OSwicGF0aCI6Ii85NzA4MjYwNy80MzI1OTYzNTAtY2U3YmViNWUtYTAyYS00ZjUzLTlkMmMtMDQyMTEyMDZjODNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDExVDA1MTQwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU0NjUzMjA1N2Q0YWZmY2IyZTFmZDVhZTQ1MWQ5ZGU3NTcwOTk0ZWRjZDAwOGFhMDQ4ODBmNDZhMDc3ZDA4NzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.qiJTUlHbcpsIDuzD6JVUkb_2TJ-1kpnCKpW346iyoic">

Sakshi-Simform commented:
> @Sakshi-Simform Have you compared your UI with the screenshot that I have provided? Just compare the UI. Can you see any difference?
> 
> Please check this comment again: [#3 (comment)](https://github.com/Sakshi-Simform/product-management/pull/3#issuecomment-2795857460)
> 
> <img alt="image" width="215" src="https://private-user-images.githubusercontent.com/97082607/432596350-ce7beb5e-a02a-4f53-9d2c-04211206c83b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQzNDg3NDksIm5iZiI6MTc0NDM0ODQ0OSwicGF0aCI6Ii85NzA4MjYwNy80MzI1OTYzNTAtY2U3YmViNWUtYTAyYS00ZjUzLTlkMmMtMDQyMTEyMDZjODNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDExVDA1MTQwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU0NjUzMjA1N2Q0YWZmY2IyZTFmZDVhZTQ1MWQ5ZGU3NTcwOTk0ZWRjZDAwOGFhMDQ4ODBmNDZhMDc3ZDA4NzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.qiJTUlHbcpsIDuzD6JVUkb_2TJ-1kpnCKpW346iyoic">

apologise I made mistake instaed of border color i changed the border top color fixing it

Sakshi-Simform commented:
> > @Sakshi-Simform Have you compared your UI with the screenshot that I have provided? Just compare the UI. Can you see any difference?
> > Please check this comment again: [#3 (comment)](https://github.com/Sakshi-Simform/product-management/pull/3#issuecomment-2795857460)
> > <img alt="image" width="215" src="https://private-user-images.githubusercontent.com/97082607/432596350-ce7beb5e-a02a-4f53-9d2c-04211206c83b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQzNDg3NDksIm5iZiI6MTc0NDM0ODQ0OSwicGF0aCI6Ii85NzA4MjYwNy80MzI1OTYzNTAtY2U3YmViNWUtYTAyYS00ZjUzLTlkMmMtMDQyMTEyMDZjODNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDExVDA1MTQwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU0NjUzMjA1N2Q0YWZmY2IyZTFmZDVhZTQ1MWQ5ZGU3NTcwOTk0ZWRjZDAwOGFhMDQ4ODBmNDZhMDc3ZDA4NzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.qiJTUlHbcpsIDuzD6JVUkb_2TJ-1kpnCKpW346iyoic">
> 
> apologise I made mistake instaed of border color i changed the border top color fixing it
fixed https://github.com/Sakshi-Simform/product-management/pull/3/commits/5b15be80d6e5238822041a0a525fb8ebd639fd70
![Screenshot from 2025-04-11 13-45-59](https://github.com/user-attachments/assets/ed39d00a-69a5-4c21-9c06-9256aaa167d9)
